api_v1/api_handler/api_variables.json

```diff
@@ -1 +1 @@
enDateRollingSum":"int","newCasesBySpecimenDateRollingRate":"float","newCasesByPublishDateDirection":"str","newCasesByPublishDateChange":"int","newCasesByPublishDateChangePercentage":"float","newCasesByPublishDateRollingSum":"int","newCasesByPublishDateRollingRate":"float","newAdmissionsDirection":"str","newAdmissionsChange":"int","newAdmissionsChangePercentage":"float","newAdmissionsRollingSum":"int","newAdmissionsRollingRate":"float","newDeaths28DaysByPublishDateDirection":"str","newDeaths28DaysByPublishDateChange":"int","newDeaths28DaysByPublishDateChangePercentage":"float","newDeaths28DaysByPublishDateRollingSum":"int","newDeaths28DaysByPublishDateRollingRate":"float","newPCRTestsByPublishDateDirection":"str","newPCRTestsByPublishDateChange":"int","newPCRTestsByPublishDateChangePercentage":"float","newPCRTestsByPublishDateRollingSum":"int","newPCRTestsByPublishDateRollingRate":"float","newVirusTestsDirection":"str","newVirusTestsChange":"int","newVirusTestsChangePercentage":"float","newVirusTestsRollingSum":"int","newVirusTestsRollingRate":"float","newCasesByPublishDateAgeDemographics":"list","newDeaths28DaysByDeathDateAgeDemographics":"list","newCasesBySpecimenDateAgeDemographics":"list","uniqueCasePositivityBySpecimenDateRollingSum":"float","uniquePeopleTestedBySpecimenDateRollingSum":"int","newPeopleReceivingFirstDose":"int","cumPeopleReceivingFirstDose":"int","newPeopleReceivingSecondDose":"int","cumPeopleReceivingSecondDose":"int","newPeopleVaccinatedFirstDoseByPublishDate":"int","cumPeopleVaccinatedFirstDoseByPublishDate":"int","newPeopleVaccinatedSecondDoseByPublishDate":"int","cumPeopleVaccinatedSecondDoseByPublishDate":"int","newPeopleVaccinatedCompleteByPublishDate":"int","cumPeopleVaccinatedCompleteByPublishDate":"int","weeklyPeopleVaccinatedFirstDoseByVaccinationDate":"int","weeklyPeopleVaccinatedSecondDoseByVaccinationDate":"int","cumPeopleVaccinatedFirstDoseByVaccinationDate":"int","cumPeopleVaccinatedSecondDoseByVaccinationDate":"int","newDailyNsoDeathsByDeathDate":"int","cumDailyNsoDeathsByDeathDate":"int","newWeeklyNsoDeathsByRegDate":"int","cumWeeklyNsoDeathsByRegDate":"int","cumWeeklyNsoDeathsByRegDateRate":"float","newCasesPCROnlyBySpecimenDateRollingSum":"int","newCasesLFDOnlyBySpecimenDateRollingRate":"float","newCasesLFDOnlyBySpecimenDate":"int","newCasesLFDConfirmedPCRBySpecimenDate":"int","newCasesLFDConfirmedPCRBySpecimenDateRollingRate":"float","cumCasesPCROnlyBySpecimenDate":"int","newCasesPCROnlyBySpecimenDateRollingRate":"float","newCasesLFDOnlyBySpecimenDateRollingSum":"int","cumCasesLFDConfirmedPCRBySpecimenDate":"int","cumCasesLFDOnlyBySpecimenDate":"int","newCasesPCROnlyBySpecimenDate":"int","newCasesLFDConfirmedPCRBySpecimenDateRollingSum":"int","cumVaccinationFirstDoseUptakeByPublishDatePercentage":"float","cumVaccinationSecondDoseUptakeByPublishDatePercentage":"float","cumVaccinationCompleteCoverageByPublishDatePercentage":"float","newVaccinesGivenByPublishDate":"int","cumVaccinesGivenByPublishDate":"int"}
```

Comment:
I'm sure this file is not used at all. I added the new metrics to it, but the file will probably be deleted soon.